<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.2m"
		durationms="73482"
		type="root"
		rid="32527d0cc788ab8"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="erdogb"
		rxversion="8.1.1.0"
		host="EC168"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="de-DE"
		dtnlanguage="de-CH"
		screenresolution="3840x1200"
		timestamp="24.07.2018 16:26:08"
		timestampiso="2018-07-24T14:26:08"
		endtime="24.07.2018 16:27:22"
		timeoutfactor="1"
		module="CalibrationAnalysis">
		<detail>
			CalibrationAnalysis
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="01:13.481"
			timeRelativeToTestModuleStartTime="01:13.482"
			timeWallClock="16:27:22"
			time="01:13.481"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
